plugin.js

```diff
@@ -82,7 +82,6 @@ function fastifyCachingPlugin (instance, options, next) {
 
   instance.decorate('cache', _options.cache)
   instance.decorate('cacheSegment', _options.cacheSegment)
-  instance.decorate('etagMaxLife', _options.etagMaxLife)
```

Comment:
@jsumners > can you take a look and confirm that this is safe to remove please ?
Currently this option is not documented and it is not actually used.

Reply:
https://github.com/fastify/fastify-caching/commit/60d71f8f29cca550d3ab54223bcf65cc5e789278

🤷‍♂️ 

This module should be rewritten to remove etag and headers support and only provide local caching.

Reply:
Hmmm in order to move this PR forward I will add back this line and the corresponding typescript definitions to avoid any breaking change for now.

I have to limit the exposure of my eyes to screens for medical reason for at least another week and a half, but I will come back at this package and rewrite it as soon as I can.